openshift/openshift-ansible-contrib
# Specify different flavors for roles

#### What does this PR do?
User can now use different flavors for different roles (servers).

#### How should this be manually tested?
1) Print out available Nova flavors (`openstack flavor list`). In `<path to inventory>/group_vars/all.yml`, uncomment line(s) with variables for specifying flavors (`openstack_<role name>_flavor`) and set the value to one of the other available flavors.
2) Run the provisioning.
3) Display servers with `openstack server list` and note ID of both a server with a changed flavor and a server with a default flavor.
4) With `openstack server show <server ID>`, display server's flavor and notice that specified roles have changed image names to those set in `all.yml`. The rest of the servers use the default flavor (defined in `openstack_default_flavor`).

#### Is there a relevant Issue open for this?
--

#### Who would you like to review this?
cc: @tomassedovic @bogdando PTAL
